### ISSUE TITLE
[Feat] Useful new-user infobar gone at second launch of GUI

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass#2442]
**Is your feature request related to a problem? Please describe.**

The GRASS GIS 8 GUI shows a very helpful infobar at initial startup:

![grass8-start](https://user-images.githubusercontent.com/1295172/174065563-32a1a481-2401-4559-889f-d0165d6b11c9.png)

This is not shown any more at a second or later start of GRASS GIS 8.

Example: a new user starts GRASS GIS, switches to single-window-GUI-mode and restarts the GUI. Then the infobar is gone.

**Describe the solution you'd like**

Always show the infobar unless the user clicks on "no longer show infobar" (or simply "hide" or likewise).

**Describe alternatives you've considered**

Have a [ ] setting in the preferences to enable/disable it.

**Comment by neteler** [comment]
@lindakarlovska Could the 'Help Text' dialogue box be changed so that the user has to tick a box like

`[ ] Don't show again`

?

**Comment by lindakarlovska** [comment]
> [@lindakarlovska](https://github.com/lindakarlovska) Could the 'Help Text' dialogue box be changed so that the user has to tick a box like
> 
> `[ ] Don't show again`
> 
> ?

Yep, this could help first-time users quite a lot. We actually received this feedback during usability testing in 2022, but I completely forgot about it. Thanks, @neteler, for bringing it up again.
